messagetypes.go

```diff
@@ -15,13 +15,16 @@ type MsgType uint8
 
 // MsgType mock enum
 const (
-	TEXTMESSAGE             MsgType = 0x1  //indicates a text message
-	IMAGEMESSAGE            MsgType = 0x2  //indicates a image message
+	TEXTMESSAGE             MsgType = 0x01 //indicates a text message
+	IMAGEMESSAGE            MsgType = 0x02 //indicates a image message
+	LOCATIONMESSAGE         MsgType = 0x10 //indicates a location message
 	AUDIOMESSAGE            MsgType = 0x14 //indicates a audio message
-	POLLMESSAGE             MsgType = 0x15 //indicates a poll message
-	LOCATIONMESSAGE         MsgType = 0x16 //indicates a location message
+	POLL_CREATE_MESSAGE     MsgType = 0x15 //indicates a poll message
+	POLL_VOTE_MESSAGE       MsgType = 0x16 //indicates a poll message
```

Comment:
I can see why underlines are more readable but I'd personally prefer to have them all in the same pattern.

Not sure how noisy it would be to change them all. Maybe as a follow-up?